source/clear-linux/get-started/virtual-machine-install/vmware-esxi.rst

```diff
@@ -1,80 +1,520 @@
+Run Clear Linux as a VMware\* ESXi guest OS
+###########################################
 
+VMware ESXi is a type 1 bare-metal hypervisor which runs directly on top of 
+server hardware.  Creation, configuration, management, and execution of virtual
+machines are done in the cloud.  For the desktop version of VMware, see 
+:ref:`vmware-player`.
 
-This section is based on *VMware vSphere 6* and explains how to use |CLOSIA|
-as ESXi Guest.
+This section explains how to run |CLOSIA| in a virtualized environment using 
+VMware ESXi 6.5 Update 1.  
 
-Ensure `Intel® Virtualization Technology`_ (Intel® VT) and
-`Intel® Virtualization Technology for Directed I/O`_ (Intel® VT-d) are both
-enabled in your BIOS/UEFI firmware configuration.
+There are 2 ways to create a |CL| :abbr:`VM (Virtual Machine)` to run in VMware:
 
-Run Clear Linux OS for Intel Architecture
-=========================================
+* `Method 1`: Install |CL| into a new VM.  This provides flexibility 
+  in configuring the VM size, partitions, initial bundles installation, etc.
+* `Method 2`: Use a ready-made VMware |CL| image.
 
-#. Download the `latest`_ |CL| **live** version
-   :file:`clear-{release-number}-live.img.xz`.
+Both methods are discussed below.
 
-#. Decompress the downloaded image. Uncompressed image size is ~ **5GB**.
+.. _download-clr-image-for-vmware:
 
-   * On Linux:
+Download the latest Clear Linux image
+=====================================
+
+Go to the |CL| `image`_ repository and download the desired type:
+
+* For `Method 1`, download an ISO installer image: `clear-<version>-installer.iso.xz`.
+* For `Method 2`, download a ready-made VMware image: `clear-<version>-basic.vmdk.xz`.
+
+For older versions, see the `releases`_ page.
+
+Although not required, it is recommended to download the corresponding 
+checksum file (designated with `-SHA512SUMS` at the end of the filename) 
+for the image in order to verify its integrity.
+
+.. _image: https://download.clearlinux.org/image
+.. _releases: https://download.clearlinux.org/releases
+
+.. _verify-clr-image-for-vmware:
+
+Verify the integrity of the downloaded image (recommended)
+==========================================================
+
+.. note::
+
+  The command-line examples below reference an `installer ISO` image file.  
+  Substitute it for the desired type of image file to verify.  
+
+* For Linux distros and macOS:
+
+  #.  Start a terminal emulator.
+  #.  Go to the directory with the downloaded file(s).
+  #.  To verify the integrity of the image and compare it to its original checksum, 
+      enter:
 
       .. code-block:: console
 
-         $ xz -d clear-{release-number}-live.img.xz
+        $ sha512sum ./clear-<version>-installer.iso.xz | diff ./clear-<version>-installer.iso.xz-SHA512SUMS -
+
+      If the checksum of the downloaded image is different than the original
+      checksum, the differences will displayed. An empty output indicates a match.
+
+* For Windows:
```

Comment:
Add the Windows instructions in a different section. Do not mix instructions for multiple OS in a single tutorial. Create a tutorial for each OS as appropriate. In this case, the verification procedure for Windows should be added in a separate small rst file under the Guides/Maintenance section and reused here. IMO